libraries/common/src/main/java/androidx/media3/common/util/CodecSpecificDataUtil.java

```diff
@@ -72,6 +78,83 @@ public static boolean parseCea708InitializationData(List<byte[]> initializationD
         && initializationData.get(0)[0] == 1;
   }
 
+  /**
+   * Parses an MPEG-4 Visual configuration information, as defined in ISO/IEC14496-2
+   *
+   * @param videoSpecificConfig A byte array containing the MPEG-4 Visual configuration information
+   *     to parse.
+   * @return A pair consisting of the width and the height.
+   */
+  public static Pair<Integer, Integer> parseMpeg4VideoSpecificConfig(byte[] videoSpecificConfig) {
+    int offset = 0;
+    boolean foundVOL = false;
+    ParsableByteArray scdScratchBytes = new ParsableByteArray(videoSpecificConfig);
```

Comment:
Not sure what "scd" means here? Can you use `strachBytes` here? Or please add a comment to explain.
Ditto for `scdScratchBits` on L105.